python/dgl/data/sbm.py

```diff
@@ -106,10 +170,14 @@ def _appendix_c(self):
             return q, p
 
     def collate_fn(self, x):
```

Comment:
We need also provide docstrings for the collate_fn.
I suppose this func is also used by users.